ex3.py

```diff
@@ -0,0 +1,45 @@
+import pickle, base64
+import os
+import datetime
+
+def rename_files(folder_name, key, d):
+    for file_name in os.listdir(path + folder_name + '/'):
+        num = file_name.split('_')
+        for i in d[key]:
+            if(str(num[1])[:-4] == str(i)):
+                my_source = path + folder_name + '/' + str(num[0]) + '_' + str(num[1])
+                unixToDatetime = datetime.datetime.fromtimestamp(int(num[0]) + int(d[key][i]))
+                my_dest = path + folder_name + '/' + str(unixToDatetime.date()) + ".npz"
+                os.rename(my_source, my_dest)
+
+
+dict_list = []
+dictionary = {} 
+# open the sixth file and append all the dicts to a dict_list.
+with (open("ex3/6/6.pkl", "rb")) as openfile:
```

Comment:
Don't put paths with rest of the code 

6 is a variable that should be outside the path string